cmd/clusterctl/internal/test/fake_objects.go

```diff
@@ -1104,7 +1104,12 @@ func SelectClusterObj(objs []runtime.Object, namespace, name string) *clusterv1.
 		}
 
 		if accessor.GetName() == name && accessor.GetNamespace() == namespace {
-			cluster := &clusterv1.Cluster{}
+			cluster := &clusterv1.Cluster{
+				TypeMeta: metav1.TypeMeta{
+					APIVersion: clusterv1.GroupVersion.String(),
+					Kind:       "Cluster",
+				},
+			}
 			if err := FakeScheme.Convert(o, cluster, nil); err != nil {
```

Comment:
Is the `FakeScheme` not populating the cluster TypeMeta?

Reply:
It seems not, but I did not investigate why